pages/_app.js

```diff
@@ -13,9 +13,7 @@ function MyApp({ Component, pageProps }) {
       <Navbar />
       <Component {...pageProps} />
       {/* CONTAINER FOR CONTENT BELOW TO TEST NAVBAR IS IN RIGHT POSITION*/}
-      <div className="mb-auto bg-green-500">
-        <h1>content</h1>
-      </div>
+      <div className="mb-auto bg-green-500"></div>
```

Comment:
I think you can just get rid of this now as we have content on master

Reply:
Will do